vapi.ts

```diff
@@ -200,14 +210,31 @@ export default class Vapi extends VapiEventEmitter {
     }
   }
 
+  private muteUnmuteLocalAudio(enable: boolean) {
+    if (this.call) {
+      this.call.setLocalAudio(enable);
+    }
+  }
+
   private onAppMessage(e?: DailyEventObjectAppMessage) {
     if (!e) return;
     try {
       if (e.data === "listening") {
         return this.emit("call-start");
+      } else if (e.data == "mute-audio") {
```

Comment:
Wait, I don't think mute-audio, unmute-audio, audio-unmuted are real events that come in through app-message. Unless I'm missing something.

I think app-message are exclusively json / text messages we send from our BE, not from Daily

Reply:
Yeah I dug through Daily’s documentation and it looks like the audio state is controlled exclusively through the DailyCall object